inbm/dispatcher-agent/dispatcher/aota/aota_command.py

```diff
@@ -283,7 +283,7 @@ def load(self) -> None:
             f'Package: {self._uri} Fetch Result: {result}')
 
         if result.status != CODE_OK:
-            raise AotaError(result.message)
+            raise AotaError(f"Unable to download docker image. Status = {result.status}, message = {result.message}.")
```

Comment:
**<h2>$\color{#FFB6C1}Enhancement: CodeRefactoring$</h2>**
**Suggestion:** Refactor the repeated error-raising logic into a separate method to reduce code duplication and improve maintainability.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        self._handle_fetch_error(result)
```